nomad/eval_endpoint.go

```diff
@@ -438,19 +438,40 @@ func (e *Eval) Delete(
 		return structs.ErrPermissionDenied
 	}
 
+	if args.Filter != "" && len(args.EvalIDs) > 0 {
```

Comment:
Should we reject this request if all servers don't support it?

Reply:
Good catch, yes.